tests/conftest.py

```diff
@@ -13,8 +13,6 @@
 
 configuration.OPTIONAL.MAIN_APP_DIR = "tests/.test_dependencies/"
 
-configuration.OPTIONAL.LOAD_MAIN_APP_AS_LIBRARY = True
```

Comment:
why this is not needed?

Reply:
This change is directly coming from the upstream repo and i merged those into my branch just to sync our repo.